docs/ref/contrib/contenttypes.txt

```diff
@@ -543,3 +543,25 @@ information.
 
     Subclasses of :class:`GenericInlineModelAdmin` with stacked and tabular
     layouts, respectively.
+
+.. module:: django.contrib.contenttypes.management
+
+Utilities
+=========
+.. versionadded:: 1.9
+
+The `remove_contenttypes` method can be used to programmatically remove stale content types. This is useful for automatic deployments where keyboard input isn't available. 
```

Comment:
wrap doc lines at 79 chars
need to use double backticks (not just single) to have an effect on the rendered output.